index.html

```diff
@@ -7,50 +7,48 @@
     <title>Commonplace Cooperative</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  
-  <header>
-    <!-- Header section -->
-    <div class="header">
-      <!-- Nav Bar -->
-        <div class="nav-bar">
-           <div class="left-side"> <!--rename elements to <nav> ?-->
-            <img id="logo" src="/images/logo.png" alt="commonplace-logo">
-            <p id="commonplace-coop">commonplace.coop</p>
+  <body>
+    <header>
+      <!-- Header section -->
+      <div class="header">
+        <!-- Nav Bar -->
+          <div class="nav-bar">
+             <div class="left-side"> <!--rename elements to <nav> ?-->
+              <img id="logo" src="/images/logo.png" alt="commonplace-logo">
+              <p id="commonplace-coop">commonplace.coop</p>
+            </div>
+            <div class="right-side">
+              <nav class="site-routes">
+                <a href="#" id="services">services</a>
+                <a href="#" id="media">media</a>
+                <a href="#" id="events">events</a>
+                <a href="#" id="about">about</a>
+                <a href="#" id="join">join</a>
+              </nav>
+            </div>
           </div>
-          <div class="right-side">
-            <nav class="site-routes">
-              <a href="#" id="services">services</a>
-              <a href="#" id="media">media</a>
-              <a href="#" id="events">events</a>
-              <a href="#" id="about">about</a>
-              <a href="#" id="join">join</a>
-            </nav>
+        </div>
+      <!-- section 1 -->
+        <div class="white-space-one">
+          <div class="tagline-one">
+            <h3 id="commonplace">COMMONPLACE</h3>
+              <div class="text-right">
+                <p id="where">where</p> <!-- -->
+                <p id="cooperation">cooperation</p>
+              </div>
           </div>
         </div>
-      </div>
-    <!-- section 1 -->
-      <div class="white-space-one">
-        <div class="tagline-one">
-          <h3 id="commonplace">COMMONPLACE</h3>
+      <div class="black-space">
+        <div class="tagline-two">
+          <h3 id="cooperative">COOPERATIVE</h3>
```

Comment:
`<hx>` elements don't just make the text bigger. They're also used semantically for parsing the information on the page. This is important for SEO on static sites and for ADA (screen readers).

If all you need is bigger text here, I recommend putting it in a `div` and styling it.

Reply:
Gotcha! I realized halfway through how much "div" should be used